contracts/core/Core.sol

```diff
@@ -678,14 +678,14 @@ contract Core is MasterCopyNonUpgradable, ConsensusModule, MosaicVersion, CoreSt
     }
 
     /**
-     * @notice Joins a validator while core is in creation state.
+     * @notice Joins a validator while core is in created state.
      *         Once the minimum number of validators joined, core is opened,
```

Comment:
```suggestion
     *         Once the minimum number of validators are joined, core is opened,
```

Reply:
**Once the minimum number of validators have joined.**
Let me know what you think on suggesstion ?